lms/djangoapps/courseware/tabs.py

```diff
@@ -177,7 +177,7 @@ class PDFTextbookTabs(TextbookTabsBase):
     A tab representing the collection of all PDF textbook tabs.
     """
     type = 'pdf_textbooks'
-    priority = None
+    priority = 201
```

Comment:
Priority is usually being set in the multiples of 10, in some cases it is being set by a difference of 1. Why is that the case? Can we make it consistent?

Reply:
It's being updated in multiple of 10 to allow external plug tabs to have control over where they fit in. 

For instance, a new custom tab might want to be placed in between the progress and dates tabs, so it can pick a priority between 21 and 39. 

However, Textbooks, pdf textbooks and html textbooks are very related tabs, so there isn't really a use case for a plugin to want to be specifically between pdf and html. Likewise, there isn't a good use case for a tab to appear between Course info and courseware tabs, since they both won't show up anyway.